spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -226,10 +228,8 @@ private int findEndpointPort(List<CoreV1EndpointPort> endpointPorts, String prim
 
 	@Override
 	public List<String> getServices() {
-		List<V1Service> services = properties.allNamespaces() ? serviceLister.list()
```

Comment:
listers are now namespace aware (this awareness comes from the new auto-configuration), so no need to check for what namespace we should use. As such, drop this code (as well as for the method `getInstances `)

Reply:
But if you are still using the old autoconfiguration don't we need the namespace?

Reply:
but I am not using the old auto-configuration anywhere, the old auto-configuration has been dropped and instead:

```
org.springframework.cloud.kubernetes.client.discovery.KubernetesClientInformerAutoConfiguration
```

if I understood your point correctly.

Reply:
And all the same beans are being created in the new auto configuration?

Reply:
that is correct. there are 3 integration tests added in this PR that prove that point, too.